src/pages/docs/api/examples/manage-test-steps-api.md

```
@@ -0,0 +1,233 @@
+> - **previousStepId**: The ID of the step after which the new step should be added. You can get it from the response of List Test Case Steps Using the REST API. 
+> - **templateId**: The ID of the NLP template that defines the action. You can get it by searching with the Search NLP Templates Using REST API.
+
+---
+
+## **Create a Test Step Using API**
+
+This endpoint allows users to create one step in a test case.
+
+|**Request Type**|**POST**|
+|---|---|
+|**Endpoint**|https://app.testsigma.com/api/v1/test_steps|
```

markdownlint-cli2 (0.17.2)

Bare URL used

(MD034, no-bare-urls)